dev/io.openliberty.restfulWS.3.1_fat/fat/src/io/openliberty/restfulWS31/fat/SeBootstrapTest.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.restfulWS31.fat;
+
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.TestServlet;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+import io.openliberty.restfulWS31.fat.sebootstrap.SebootstrapTestServlet;
+
+@RunWith(FATRunner.class)
+public class SeBootstrapTest extends FATServletClient {
+    private static final String war = "sebootstrap";
+
+    @Server("sebootstrap")
+    @TestServlet(servlet = SebootstrapTestServlet.class, contextRoot = war)
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        ShrinkHelper.defaultDropinApp(server, war, "io.openliberty.restfulWS31.fat.sebootstrap")
+                    .addAsWebInfResource(new StringAsset("This is a resource file that is part of the app"),
+                                         "resource.txt");
+
+        // Make sure we don't fail because we try to start an
+        // already started server
+        try {
+            server.startServer(true);
+        } catch (Exception e) {
+            System.out.println(e.toString());
```

Comment:
Why catch the exception instead of letting it fail the setUp?

Reply:
The test I initially copied was attempting to prevent a failure if the server was already started.   I see no point in continuing this for this tests and have removed.

Reply:
Done.